### PR TITLE
Add verifier to the cli

### PR DESCRIPTION
**Pull request by JordyRo1**

### CODE REVIEW

**Comment by JordyRo1** [comment]
Task is complete. However, the public key defined in the function verify_random must be compatible with the ecvrf, which is different from the current public key. In order to find the actual public key for the ecvrf, just call the `get_public_key(sk)` from the randomness_utils.py. `sk` represents the private key associated with the account used for the deployment of the randomness contract. Once the public key is derived, you can just convert it into a bytes object and replace the current `pub_key` variable defined in the verify_random function. To make sure the process is successful, you can just run the verify_random function with this transaction hash: `0x0302c9274be9f3e10c480a8cdfc1015cd9f0e14a9c16b90e2aebea7ac2e215e1`, which is a tx hash for the mainnet contract, and the result should be valid.

File: cli/random.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    # Define the 'verify-random' subcommand
+    verify_random_parser = random_subparsers.add_parser("verify-random")
+    verify_random_parser.add_argument(
+        "transaction_hash", type=str, help="Transaction hash"
+    )
+    verify_random_parser.add_argument(
+        "--network",
+        type=str,
+        default="sepolia",
```
Comment (EvolveArt) [inline cli/random.py:23]: mainnet ?

File: pragma/core/types.py [hunk 2]
```diff
@@ -69,10 +73,15 @@
     SEPOLIA: [
         "https://starknet-sepolia.public.blastapi.io/rpc/v0_6",
     ],
+    SEPOLIA: [
+        # "https://starknet-sepolia.public.blastapi.io"
+        f"https://rpc.nethermind.io/sepolia-juno/?apikey={os.getenv('RPC_SEPOLIA_KEY')}"
+    ],
```
Comment (EvolveArt) [inline pragma/core/types.py:79]: duplicate

File: pragma/core/types.py [hunk 3]
```diff
@@ -69,10 +73,15 @@
...
 }
 
 
 def get_rpc_url(network=TESTNET, port=5050):
+    print(os.getenv("RPC_SEPOLIA_KEY"))
```
Comment (EvolveArt) [inline pragma/core/types.py:84]: remove

**Comment by akhercha** [comment]
We should close this PR as it's outdated now, right @JordyRo1 ?

**Pull request closed without merging**